addons/hr_recruitment_survey/models/survey_invite.py

```diff
@@ -1,22 +1,23 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import fields, models, _
+from odoo.tools.misc import clean_context
 
 
 class SurveyInvite(models.TransientModel):
     _inherit = "survey.invite"
 
-    applicant_id = fields.Many2one('hr.applicant', string='Applicant', default=lambda self: self.env.context.get('active_id', None))
+    applicant_id = fields.Many2one('hr.applicant', string='Applicant')
 
     def action_invite(self):
         self.ensure_one()
 
-        if not self.applicant_id.response_id:
-            response = self.applicant_id.survey_id._create_answer(partner=self.applicant_id.partner_id)
-            self.applicant_id.response_id = response.id
-
-        body = _('The survey has been sent to "%s".', self.applicant_id.partner_name)
-        self.applicant_id.message_post(body=body)
+        if self.applicant_id:
+            if not self.applicant_id.response_id:
```

Comment:
Could probably merge those 2 "if" right?

Reply:
I don't know, depends if we want to log that  'survey has been sent to the applicant' as many times as the recruiter send the interview.. then makes sense to have two if statements. 

Even though it does not make sense functionally to 'Send Interview' more then once, if we click 5 times on it, 5 mails will be sent. So, I think it somehow makes sense to have 5 logs about it in chatter.

What do you think?

Reply:
I did not notice the indentation was different on those two lines so nevermind :smile: 

(Although it's a bit strange that we log "survey has been sent to ..." when potentially nothing really happened because we already sent it, but that's another story, maybe an improvement for master later).